lib/authentication-context.js

```diff
@@ -142,6 +142,12 @@ Object.defineProperty(AuthenticationContext.prototype, 'options', {
   }
 });
 
+Object.defineProperty(AuthenticationContext.prototype, 'extraQP', {
```

Comment:
The extra query parameters you see in the other platforms only send it to the authorize endpoint as far as I know.  No platform sends it to the token endpoint.